tests/e2e/classic_ingress_test.go

```diff
@@ -80,6 +102,250 @@ var _ = Describe("Classic Ingress", ci.FeatureIngress, func() {
 			Expect(string(ingress.LoadBalancerType())).To(Equal("nlb"))
 		})
 
+	It("update ingress components - [id:73610]",
+		ci.Day2,
```

Comment:
Split the validation checkpoints to another negative medium case.

Reply:
@xueli181114 Sorry it took so long to get back to you. But, what label(s) should I be using for the negative test case? The only one I see for negative test cases is `Day1Negative`, is that what I should use?

Reply:
@xueli181114 I split the negative tests into another test case: https://polarion.engineering.redhat.com/polarion/#/project/OSE/workitem?id=OCP-75067. Can you take a look at it, as well as take another look at this PR?

Reply:
@jerichokeyne The labels.Medium generally means negative validation cases.